README.md

```diff
@@ -4,79 +4,79 @@
 
 This release focus on the following:
 
-- We try to improve build performance with Persistent Caching.
-- We try to improve Long Term Caching with better algorithms and defaults.
-- We try to cleanup internal structures that were left in a weird state while implementing features in v4 without breaking changes.
-- We try to prepare for future features by doing breaking changes now to stay on v5 as long as possible.
+- We try and improve build performance with Persistent Caching.
```

Comment:
Can you explain the change here?

Reply:
I changed this before realising it is the opposite, and forgot to delete (was a long commit after all)
try and is mainly used for friendly conversations, whereas try to is the correct approach.
Will change.

Reply:
reverted